runtime/parachains/src/paras_inherent.rs

```diff
@@ -1217,12 +1220,12 @@ mod tests {
 			new_test_ext(MockGenesisConfig::default()).execute_with(|| {
 				// Create the inherent data for this block
 				let dispute_statements = BTreeMap::new();
-				// No backed and concluding cores, so all cores will be fileld with disputesw
+				// No backed and concluding cores, so all cores will be filld with disputes.
```

Comment:
If this was `cargo-spellcheck` not doing the correcting right, please file an issue there.